links.html

```diff
@@ -1,74 +1,85 @@
-			 Class Links</h1>
+			<h1> <br>Intro to HTML/CSS:<br>
+				Class Links</h1>
 			</header>
 
 
 			<section class="lists">
-			
-			<div class="column">
-			
-			<h2>Class 1: Angle Brackets</h2>
-			<ul>
-			<li><a href="http://www.catherinerust.com/intro/class1.html#/" target="_blank">Class 1 Slides</a></li>
-			<li><a href="http://www.catherinerust.com/intro/homework1.html" target="_blank">Class 1 Homework</a></li>
-			<li><a href="" target="_blank">Class 1 Worksheet</a></li>
-			</ul>
-			
-			</div>
-
-			<div class="column">
-			
-			<h2>Class 2: Curly Braces</h2>
-			<ul>
-			<li><a href="http://www.catherinerust.com/intro/class2.html#/" target="_blank">Class 2 Slides</a></li>
-			<li><a href="http://www.catherinerust.com/intro/homework2.html" target="_blank">Class 2 Homework</a></li>
-			<li><a href="" target="_blank">Class 2 Worksheet</a></li>
-			</ul>
-			
-			</div>
-
-			<div class="column">
-			
-			<h2>Class 3: </h2>
-			<ul>
-			<li><a href="http://www.catherinerust.com/intro/class3.html#/" target="_blank">Class 3 Slides</a></li>
-			<li><a href="http://www.catherinerust.com/intro/homework3.html" target="_blank">Class 3 Homework</a></li>
-			<li><a href="" target="_blank">Class 3 Worksheet</a></li>
-			</ul>
-			
-			</div>
-
-			<div class="column">
-			
-			<h2>Class 4: Positioning</h2>
-			<ul>
-			<li><a href="http://www.catherinerust.com/intro/class4.html#/" target="_blank">Class 4 Slides</a></li>
-			<li><a href="" target="_blank">Class 4 Worksheet</a></li>
-			</ul>
-			
-			</div>
-			
+
+				<div class="column">
+
+					<h2>Class 1: Angle Brackets</h2>
+					<ul>
+						<li><a href="http://www.catherinerust.com/intro/class1.html#/" target="_blank">Class 1 Slides</a></li>
+						<li><a href="http://www.catherinerust.com/intro/homework1.html" target="_blank">Class 1 Homework</a></li>
+						<li><a href="worksheet1.pdf" target="_blank">Class 1 Worksheet</a></li>
+						<li><a href="skeleton.html" target="_blank">HTML Skeleton</a></li>
+						<li><a href="https://validator.w3.org/#validate_by_input" target="_blank">W3 Markup Validator</a></li>
+						<li><a href="http://www.w3.org/" target="_blank">W3C</a></li>
+					</ul>
+
+
+				</div>
+
+				<div class="column">
+
+					<h2>Class 2: Curly Braces</h2>
+					<ul>
+						<li><a href="http://www.catherinerust.com/intro/class2.html#/" target="_blank">Class 2 Slides</a></li>
+						<li><a href="http://www.catherinerust.com/intro/homework2.html" target="_blank">Class 2 Homework</a></li>
+						<li><a href="worksheet2.pdf" target="_blank">Class 2 Worksheet</a></li>
+					</ul>
+
+				</div>
+
+				<div class="column">
+
+					<h2>Class 3: More CSS</h2>
+					<ul>
+						<li><a href="http://www.catherinerust.com/intro/class3.html#/" target="_blank">Class 3 Slides</a></li>
+						<li><a href="http://www.catherinerust.com/intro/homework3.html" target="_blank">Class 3 Homework</a></li>
+						<li><a href="worksheet3.pdf" target="_blank">Class 3 Worksheet</a></li>
```

Comment:
Change both URLs to reflect the `http://catheraaine.com/intro/class3.html#/` domain